packages/eslint-plugin-pf-codemods/lib/rules/v5/datalist-remove-selectableRow.js

```diff
@@ -0,0 +1,25 @@
+const { renameProp, getPackageImports} = require('../../helpers');
+
+//https://github.com/patternfly/patternfly-react/pull/8827
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const dataListImport =  getPackageImports(context, '@patternfly/react-core')
+      .filter(specifier => specifier.imported.name == 'DataList');
+
+    return dataListImport.length === 0 ? {} : {
+      JSXOpeningElement(node) {
+        if (dataListImport.map(imp => imp.local.name).includes(node.name.name)) {
+          const selectableRow = node.attributes.find(a => a.name?.name === 'selectableRow');
+          if (selectableRow) {
+            context.report({
+              node,
+              message: `DataList's selectableRow.onChange property has been replaced with onSelectableRowChange. The order of the params in the callback has also been updated so that the event param is first.`,
```

Comment:
Nit: Would saying "DataList's selectableRow prop has been replaced with the onSelectableRowChange callback" be more clear?

Reply:
maybe - I wondered how to succinctly convey that the prop name AND type has updated - as well as the param order in the callback. but I may not be giving enough credit to the consumer to be able to figure it out.